exes/mqtt-bridge/tests/inc/endpoint_mock.hpp

```diff
@@ -66,22 +66,32 @@ class endpoint_client_mock {
 
   auto strand() -> asio::strand<asio::io_context::executor_type>& { return strand_; }
 
-  auto recv(async_mqtt::control_packet_type packet_t) -> asio::awaitable<
-      package_v<async_mqtt::v5::suback_packet, async_mqtt::v5::publish_packet, async_mqtt::v5::connack_packet>> {
+  using package_v_mock = package_v<async_mqtt::v5::suback_packet,
+                                   async_mqtt::v5::publish_packet,
+                                   async_mqtt::v5::connack_packet,
+                                   async_mqtt::v5::puback_packet,
+                                   async_mqtt::v5::pubcomp_packet,
+                                   async_mqtt::v5::pubrec_packet,
+                                   async_mqtt::v5::pubrel_packet,
+                                   async_mqtt::v5::subscribe_packet,
+                                   async_mqtt::v5::unsuback_packet,
+                                   async_mqtt::v5::unsubscribe_packet>;
+
+  auto recv(async_mqtt::control_packet_type packet_t) -> asio::awaitable<package_v_mock> {
     if (packet_t == async_mqtt::control_packet_type::suback) {
-      package_v<async_mqtt::v5::suback_packet, async_mqtt::v5::publish_packet, async_mqtt::v5::connack_packet> my_variant;
+      package_v_mock my_variant;
```

Comment:
This variable is used in all cases. Please just define it above the conditionals.